### ISSUE TITLE
Question values of string type are converted to float

### ISSUE THREAD

**Issue by Tejeev** [rancher/rancher#27801]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**

Rancher parses strings input as Question Values that only contain numbers as a float. For example, using this chart with the supplied values and questions:

A deployment that set environment variables:
```
        env:
            - name: TEST_VALUE
              value: "{{ .Values.testValue }}"
            - name: TEST_QUESTION_VALUE
              value: "{{ .Values.testQuestionValue }}"
```

values.yaml:
```
testValue: "1234567890"
testQuestionValue: "1234567890"
```

questions.yaml:
```
- variable: testQuestionValue
    label: Test Question Value
    type: string
    default: "1234567890"
    required: true
    description: "Test question to demonstrate unexpected behavior"
    group: "OTP Adapter"
```

The output of the environment variables show the question value as a float:
env:
- name: TEST_VALUE
value: "1234567890"
- name: TEST_QUESTION_VALUE
value: "1.23456789e+09"

Attached are the sample snipets with screenshots of the issue.  This was tested in the Rancher UI, and in the CLI:
```
$ ./rancher app install basic-helm test
Description: Test question to demonstrate unexpected behavior
Name: Test Question Value
Variable Name: testQuestionValue
Default:[1234567890]
Enter answer or 'return' for default:123454321
run "app show-notes test" to view app notes once app is ready
```

**Result:**
From UI tests:
<img width="608" alt="rancher-questions" src="https://user-images.githubusercontent.com/8906089/86292099-ffb34680-bbe7-11ea-9103-992f2da9c154.png">
<img width="257" alt="templated-deployment" src="https://user-images.githubusercontent.com/8906089/86292119-0fcb2600-bbe8-11ea-9824-24ee9d293a7d.png">

CLI tests:
![image (5)](https://user-images.githubusercontent.com/8906089/86292140-1f4a6f00-bbe8-11ea-9c90-cfd1a61c2fe3.png)

gz#10874

**Comment by sowmyav27** [comment]
Reproduced the issue on 2.4.5 and **2.3.8**

1. Deploy mysql app
2. admin/password - give value `12345678921`
3. Deploy the app.
4. Navigate to workload --> mysql --> Environment variables

<img width="1310" alt="Screen Shot 2020-07-01 at 5 22 43 PM" src="https://user-images.githubusercontent.com/26032343/86304510-e4cedb00-bbc3-11ea-8847-8c417c3b1be3.png">

**Comment by jedrivisser** [comment]
Example helm chart that reproduces this issue: https://github.com/jedrivisser/basic-helm

**Comment by PennyScissors** [comment]
Closing this because the original JIRA issue for this SURE-2197 was resolved.

**Comment by jedrivisser** [comment]
@PennyScissors could you maybe paste some details from that ticket here? How was it resolved? What version etc?

**Comment by PennyScissors** [comment]
Hello @jedrivisser, it was marked resolved because we couldn't reproduce it in the Apps & Marketplace of Rancher v2.6 following the steps below. Although it is reproducible in the old catalog provided by the Cluster Manager UI available in Rancher v2.4 and v2.5, those Rancher versions have now reached EOL and EOM (See [here](https://www.suse.com/lifecycle/)) and are no longer getting fixes. I apologize for the inconvenience.

Steps taken in Rancher v2.6.4
1. Add custom catalog to Apps & Marketplace using `https://github.com/jedrivisser/basic-helm`
2. Deploy `basic-helm` chart with default values:
```
testQuestionValue: '1234567890'
testValue: '1234567890'
```
3. Verify values were not changed in the chart's config map
![image](https://user-images.githubusercontent.com/8951271/164289127-a0b9304b-7158-4f86-bda0-97a0913f573f.png)

**Comment by jedrivisser** [comment]
Great, thanks!
